# エラーステータス(旧ロボットベースのみ)GET
---
uint8  adc
uint8  com
uint8  i2c0
uint8  i2c1

